cluster/gce/list-resources.sh

```diff
@@ -37,7 +37,21 @@ GREP_REGEX=""
 function gcloud-compute-list() {
   local -r resource=$1
   echo -e "\n\n[ ${resource} ]"
-  gcloud compute ${resource} list --project=${PROJECT} ${@:2} | grep "${GREP_REGEX}"
+  local attempt=1
+  local result=""
+  while true; do
+    echo "Attempt ${attempt} to list ${1} in GCE"
+    if result=$(gcloud compute ${resource} list --project=${PROJECT} ${@:2} | grep "${GREP_REGEX}"); then
+      return ${result}
```

Comment:
this should be `echo ${result}` (possibly with -e? not sure how newlines are treated in L44) followed by `return`